src/main/java/io/clue2solve/aws/bedrock/springboot/starter/config/LlamaProperties.java

```diff
@@ -9,12 +9,26 @@
 import java.util.List;
 import java.util.Optional;
 
+/**
+ * Sample properties aws.bedrock.model.llama2.modelId=meta.llama2-13b-chat-v1
+ * aws.bedrock.model.llama2.topP=0.9 aws.bedrock.model.llama2.temperature=0.5
+ * aws.bedrock.model.llama2.maxTokens=100
+ *
+ * @see <a href="https://docs.ai21.com/reference/llama2-complete-api-ref">...</a>
```

Comment:
Not correct or valid hyperlink.  Consider https://docs.aws.amazon.com/bedrock/latest/userguide/model-parameters-meta.html.